src/components/index.ts

```diff
@@ -263,6 +263,8 @@ export {
   DEFAULT_TOAST_DURATION,
 } from './Toast';
 
+export {default as Toggle} from './Toggle';
```

Comment:
We should be exporting props here as well.

```suggestion
export {default as Toggle, Props as ToggleProps} from './Toggle';
```